aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/RealAWSCognitoAuthPlugin.kt

```diff
@@ -1516,7 +1576,11 @@ internal class RealAWSCognitoAuthPlugin(
                             authZState is AuthorizationState.StoringCredentials -> {
                                 credentialStoreStateMachine.send(
                                     CredentialStoreEvent(
-                                        CredentialStoreEvent.EventType.StoreCredentials(authZState.amplifyCredential)
+                                        CredentialStoreEvent.EventType.StoreCredentials(
+                                            AmplifyCredentialType.AMPLIFY_CREDENTIAL,
+                                            null,
```

Comment:
same here

Reply:
We removed the nullable property and now using sealed data classes with properties.